Integrations/integration-UrlScan.yml

```diff
@@ -38,10 +44,10 @@ script:
     requests.packages.urllib3.disable_warnings()
 
 
-
     '''GLOBAL VARS'''
     BASE_URL = 'https://urlscan.io/api/v1/'
     APIKEY = demisto.params().get('apikey')
+    THRESHOLD = int(demisto.params().get('url_threshold', None)) or 1;
```

Comment:
why not `int(demisto.params().get('url_threshold', '1'))`?
this way `0` is not  a possible threshold because you will get `1`